### ISSUE TITLE
[Bug]

### ISSUE THREAD

**Issue by theweeeknd** [JasonStein/Notepads#1064]
卡在如图这里，1min后自动退出
**重置应用数据**可**暂时**恢复正常，但没多久（<1天？）问题会重现
Notepads 是我最高频使用的app之一，但之前从没遇过这种 _重置数据都不能彻底解决_ 的诡异bug？

**Screenshots**
![image](https://user-images.githubusercontent.com/84619641/159411927-7cc30929-9e6f-417d-80d4-d223a78a240e.png)

Desktop (please complete the following information):
**OS: [Windows 11 21H2 22000.556]
Version [1.4.7.0]**

**Comment by JasonStein** [comment]
你之前有打开过什么大文件吗？每一次出现这种情况之前有没有特例？上一次关闭的时候是否有很多tab？然后出现这种情况是恢复休眠之后还是？

**Comment by Pitu** [comment]
I'm having the same issue using the latest version on Windows 10 20H2 (OS Build 19042.1586).
As a bit of additional information in my case, the Windows Store process is blocked by a firewall and it's impossible for it to connect to the internet. Not sure if this would affect notepads but it was working just fine until the last update.

Now whenever I try opening a file, or open notepads.exe from the start menu I get the screen in the screenshot above and there's nothing else I can do.
If there are any logs I can supply to help debug this I'd be glad to, if you can tell me where to find them.

**Comment by JasonStein** [comment]
> I'm having the same issue using the latest version on Windows 10 20H2 (OS Build 19042.1586). As a bit of additional information in my case, the Windows Store process is blocked by a firewall and it's impossible for it to connect to the internet. Not sure if this would affect notepads but it was working just fine until the last update.
> 
> Now whenever I try opening a file, or open notepads.exe from the start menu I get the screen in the screenshot above and there's nothing else I can do. If there are any logs I can supply to help debug this I'd be glad to, if you can tell me where to find them.

Seems suspicious, since all store apps do connect to Store directly for analysis purposes (out of my control). Another thing that might cause this issue is the AppCenter which I was using in this app for recording app analysis. Try unblocking the firewall first and see if it helps. Since this is not a crash, I don't think event viewer will be helpful in this case.

**Comment by theweeeknd** [comment]
> 你之前有打开过什么大文件吗？每一次出现这种情况之前有没有特例？上一次关闭的时候是否有很多tab？然后出现这种情况是恢复休眠之后还是？

都没有，我感觉没发现任何规律

另补充些线索：

应用和功能 - 终止 
_无用_

应用和功能 - 修复
_无用_

应用和功能 - 重置
_暂时正常，1天内问题会复现_

应用和功能 - 卸载
_同 重置_

应用和功能 - 后台应用权限
_无用_

关闭 快照备份与还原
_无用_

**Comment by JasonStein** [comment]
你平时有用别的UWP app吗？比如windows的mail或者其他的？

**Comment by theweeeknd** [comment]
> 你平时有用别的UWP app吗？比如windows的mail或者其他的？

应该就notepads一个…

**Comment by sameer-coder** [comment]
Facing the same issue after installing the latest version from the releases page. I don't have any antivirus or firewall on my system. Is there any fix for this yet? Thanks

**Comment by xiaoyv404** [comment]
我也是这样，摊手

**Comment by xiaoyv404** [comment]
> 我也是这样，摊手

[v1.4.7.0](https://github.com/0x7c13/Notepads/releases/tag/v1.4.7.0)以前都正常，就1.4.70开始有问题了

**Comment by sameer-coder** [comment]
> 你平时有用别的UWP app吗？比如windows的mail或者其他的？

@0x7c13  Yes, I do have multiple UWP apps installed and they are working fine. Notepads just starts and shows the blanks screen and you can't do anything. I don't have any firewalls or antivirus blocking it. I have blocked windows update though but other UWP apps still work fine. Not pushing anyone to fix this urgently but would have really liked to try out this app.

![image](https://user-images.githubusercontent.com/2347454/168465275-17c7a06e-79cf-4307-986b-490756259e8f.png)

**Comment by xiaoyv404** [comment]
> > 你平时有用别的UWP app吗？比如windows的mail或者其他的？
> 
> @0x7c13 Yes, I do have multiple UWP apps installed and they are working fine. Notepads just starts and shows the blanks screen and you can't do anything. I don't have any firewalls or antivirus blocking it. I have blocked windows update though but other UWP apps still work fine. Not pushing anyone to fix this urgently but would have really liked to try out this app.
> 
> ![image](https://user-images.githubusercontent.com/2347454/168465275-17c7a06e-79cf-4307-986b-490756259e8f.png)

You can try this release [v1.4.7.0](https://github.com/0x7c13/Notepads/releases/tag/v1.4.7.0) it can run

Please forgive my bad English :)

**Comment by user95f85f** [comment]
Temporary fix on Windows 10:
search by hitting your windows key
search "App"
click "Add or remove programs"
scroll down to Notepads
left click it
left click "Advanced options"
scroll down & hit the "Reset" button
now Notepads should work.

Note to developer:
C:\Users\MyUsername\AppData\Local\Packages\19282JackieLiu.Notepads-Beta_echhpq9pdbte8\LocalState\Microsoft.AppCenter.Storage
SQLite table file I'm 3% sure is the problem with Notepads not working.

Version I'm using: 1.4.7 on the Windows 10 Store.
My crashing happened randomly. Not on an update. The crashing was ever-persistent. The problem only happened once. I've been using Notepads quite a lot. (like at least 80 launches within a 10-20 day period). I'm using the latest Windows 10 (the 2H20 one or whatever).

**Comment by 0x7c13** [comment]
> Temporary fix on Windows 10: search by hitting your windows key search "App" click "Add or remove programs" scroll down to Notepads left click it left click "Advanced options" scroll down & hit the "Reset" button now Notepads should work.
> 
> Note to developer: C:\Users\MyUsername\AppData\Local\Packages\19282JackieLiu.Notepads-Beta_echhpq9pdbte8\LocalState\Microsoft.AppCenter.Storage SQLite table file I'm 3% sure is the problem with Notepads not working.
> 
> Version I'm using: 1.4.7 on the Windows 10 Store. My crashing happened randomly. Not on an update. The crashing was ever-persistent. The problem only happened once. I've been using Notepads quite a lot. (like at least 80 launches within a 10-20 day period). I'm using the latest Windows 10 (the 2H20 one or whatever).

I also have a feeling that this might be caused by AppCenter. I will update the version of the AppCenter and create a new release to see if it will solve the problem for you guys.

**Comment by 0x7c13** [comment]
![image](https://user-images.githubusercontent.com/1166162/169886278-bc9ed39c-5bf5-4c42-a5d7-9705e067a179.png)

**Comment by 0x7c13** [comment]
1.4.8.0 is released today, please help verify if the issue is mitigated.

**Comment by sameer-coder** [comment]
> 1.4.8.0 is released today, please help verify if the issue is mitigated.

I can't seem to find 1.4.8.0 release. Latest I can see is 1.4.7.0 🤔

**Comment by 0x7c13** [comment]
> > 1.4.8.0 is released today, please help verify if the issue is mitigated.
> 
> I can't seem to find 1.4.8.0 release. Latest I can see is 1.4.7.0 🤔

https://github.com/0x7c13/Notepads/releases/tag/v1.4.8.0

**Comment by sameer-coder** [comment]
> 1.4.8.0 is released today, please help verify if the issue is mitigated.

Thanks for the effort. Unfortunately the issue persists :(

**Comment by 0x7c13** [comment]
> > 1.4.8.0 is released today, please help verify if the issue is mitigated.
> 
> Thanks for the effort. Unfortunately the issue persists :(

Remove Notepads from your computer, go to release page and download 1.4.6.0 and see if it works: https://github.com/0x7c13/Notepads/releases/tag/v1.4.6.0

**Comment by user95f85f** [comment]
> Thanks for the effort. Unfortunately the issue persists :(
Huh. Are you using the Windows Store version?
If you installed the .msi from github release have you tried uninstalling
the Windows Store version?
If you are using the Windows Store version there are some special Windows
Store stuff you can try:
microsoft store cache reset:
1) open up admin cmd
2) WSReset.exe

If that doesn't work you can go full crazy and see if your Windows PC has
serious problems (do this in admin cmd.exe) one after another:

SFC /SCANNOW
DISM /ONLINE /CLEANUP-IMAGE /RESTOREHEALTH

I got these commands from a Epic Games Troubleshooting Help Email
because Fortnite would stop downloading in the Epic Games Store. (It
didn't fix my problem and the commands basically did nothing).

**Comment by user95f85f** [comment]
> Thanks for the effort. Unfortunately the issue persists :(
OK I think I have a potential solution:
1) open up task manager (WIN+X and select Task Manager is the fastest way
to open it IMO)
2) end all "Notepads" processes.
3) now try to Reset the Notepads App in the Add and Remove Programs ->
Notepads -> Advanced Settings page.
4) now try starting Notepads.
